app/feature/feature-delete-account/src/main/kotlin/com/hedvig/android/feature/deleteaccount/DeleteAccountDestination.kt

```diff
@@ -102,20 +99,16 @@ private fun DeleteScreenContents(
   description: String,
   buttonText: String,
   onButtonClick: () -> Unit,
+  isButtonRed: Boolean,
   modifier: Modifier = Modifier,
   isButtonLoading: Boolean = false,
-  buttonColors: ButtonColors = ButtonDefaults.buttonColors(
-    containerColor = MaterialTheme.colorScheme.primary,
-    contentColor = MaterialTheme.colorScheme.onPrimary,
-    disabledContainerColor = MaterialTheme.colorScheme.primary.copy(alpha = 0.12f),
-    disabledContentColor = MaterialTheme.colorScheme.onPrimary.copy(alpha = 0.38f),
-  ),
 ) {
   Column(modifier) {
     Spacer(Modifier.height(16.dp))
-    Text(
+    HedvigText(
       text = title,
-      style = MaterialTheme.typography.headlineMedium,
+      textAlign = TextAlign.Center,
```

Comment:
A lot of our newer screens that have this top attached text now are left-aligned. Do we want this one to be center aligned here? It seems like it was not as such before.